timely_beliefs/beliefs/classes.py

```diff
@@ -1678,21 +1678,23 @@ def set_reference_values(
         reference_belief_horizon: timedelta = None,
         reference_source: BeliefSource = None,
         return_expected_value: bool = False,
+        return_middle_value: bool = False,
     ) -> "BeliefsDataFrame":
         """Add a column with reference values.
-        By default the reference will be the expected value of the most recent belief held by the same source.
+        By default, the reference will be the probabilistic value of the most recent belief held by the same source.
+        To set a deterministic reference, use either return_expected_value or return_middle_value.
         Optionally, set a reference belief horizon.
         Optionally, set a reference source present in the BeliefsDataFrame.
-        These options allow to define what is considered to be true at a certain time after an event.
-        Todo: Option to set probabilistic reference values
+        These options allow defining what is considered to be true at a certain time after an event.
 
         :param reference_belief_time: optional datetime to indicate that
                the accuracy should be determined with respect to the latest belief held at this time
         :param reference_belief_horizon: optional timedelta to indicate that
                the accuracy should be determined with respect to the latest belief at this duration past knowledge time
         :param reference_source: optional BeliefSource to indicate that
                the accuracy should be determined with respect to the beliefs held by the given source
-        :param return_expected_value: if True, set a deterministic reference by picking the expected value
+        :param return_expected_value: if True, set a deterministic reference by picking the mean value
+        :param return_middle_value: if True, set a deterministic reference by picking the median value
```

Comment:
Here (and maybe in `set_reference` as well) we should document that you need to pick one of them, otherwise you run into an error at runtime.

Reply:
`set_reference_values` can be called without picking either, in which case a probabilistic reference is returned. For example, see the `accuracy` method in `beliefs/classes.py`. Fun fact: timely beliefs is actually able to compute accuracy metrics again probabilistic references (something which I have yet to encounter even in literature).

If you actually ran into an error, please let me know what it was.

Reply:
Sorry, I meant that you cannot pick both.

Reply:
Ah, I see. Thanks. I took the opportunity to switch to a different API for these arguments.